src/server.js

```diff
@@ -133,6 +148,9 @@ function jsonExporter(req, res, next) {
             title,
             description,
             review_state,
+            ...(preview_image_link['@id'] !== '' && {
```

Comment:
@sneridagh 

its because the preview image in content creator is just a string of the path to the image, instead of an object with "@id"

If we just copy over the whole object, like for the others, then it won't work, we have to extract the path from the "@id"

Reply:
@sneridagh @ThomasKindermann It uses the normal serializer/deserializer for a RelationChoice field. I think the deserializer accepts multiple formats including a string path, an object with an `@id`, or a uuid. I chose to use the path when I updated the content creator files since it's simplest for human editors.